Modules/Sources/UITestsFoundation/Screens/ReaderScreen.swift

```diff
@@ -143,8 +143,7 @@ public class ReaderScreen: ScreenObject {
     private func waitForLoadingToFinish() {
         let doesNotExistPredicate = NSPredicate(format: "exists == FALSE")
         let expectation = XCTNSPredicateExpectation(predicate: doesNotExistPredicate, object: ghostLoading)
-        let result = XCTWaiter.wait(for: [expectation], timeout: 5.0)
-        XCTAssertEqual(result, .completed)
+        let _ = XCTWaiter.wait(for: [expectation], timeout: 5.0)
```

Comment:
This should work, too, right?

```suggestion
        _ = XCTWaiter.wait(for: [expectation], timeout: 5.0)
```

FWIW, I prefer it because it looks odd to me to define a `let` or `var` as unused.

Reply:
Yes. Thanks for the suggestion.